src/styles/blocks/catalog.scss

```diff
@@ -11,6 +11,7 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    transition: $time;
```

Comment:
It is better to add property for transition, like transition: color $time; (where color - is property which was changed)